### DISCUSSION TITLE
Integration of ERP Functionalities

### DISCUSSION THREAD

**Discussion by AndryHTC** [twentyhq/twenty#1053] [Ideas]
Hello Twenty team,

Firstly, I would like to commend your efforts on creating an open-source CRM solution that's not only user-friendly but also highly extendable.

I'm curious about the possibility of integrating ERP functionalities into Twenty. As it stands, there's a noticeable gap in the market for a robust, open-source CRM/ERP system. The integration of ERP functionalities such as **accounting/invoices, warehouse management, and transport** could make Twenty an even more powerful tool for businesses.

Are there any plans to add such features in the future? I believe this could greatly enhance the utility and scope of Twenty, making it a one-stop solution for businesses.

Looking forward to hearing your thoughts.

**Comment by FelixMalfait** [comment]
I think building a full-fledged ERP would be too much work as building a great CRM is already A LOT of work. So I don't see us building modules such as warehouse management into the core codebase anytime soon.
BUT... I agree we should provide a way for community members to create apps, and we will likely within the next ~6months. So if someone want to work on a warehouse app then they can build it and share it with the community!
